packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/FrameCarousel.js

```diff
@@ -49,73 +49,108 @@ export function NextLabel () {
   )
 }
 
-export const StyledInput = styled.input`
+export const StyledFrame = styled.label`
   ${props => props.theme && css`
-    &:checked + img {
-      outline: ${props.theme.global.colors['neutral-4']} solid;
+    input:checked + img {
+      border: ${props.theme.global.colors['neutral-4']} solid;
+    }
+    
+    input:focus + img {
+      outline: 2px solid ${tint(0.5, props.theme.global.colors.brand)};
+    }
+
+    input:hover + img {
+      outline: 2px solid ${tint(0.5, props.theme.global.colors.brand)};
     }
   `}
-  clip: rect(0 0 0 0);
-  overflow: hidden;
-  position: absolute;
+  input {
+    opacity: 0.01;
+    position: absolute;
+  }
+
 `
 
 export const StyledImage = styled.img`
-  height: 3em;
-  width: 3em;
-  margin: 0.5em;
-  float: center;
+  height: 40px;
   object-fit: cover;
   padding: 0;
+  width: 40px;
 `
 
 class FrameCarousel extends React.Component {
   constructor () {
     super()
+    this.handleScroll = this.handleScroll.bind(this)
     this.handlePrevious = this.handlePrevious.bind(this)
     this.handleNext = this.handleNext.bind(this)
+    this.activeLabel = React.createRef()
+    this.frameList = React.createRef()
+  }
+
+  componentDidMount () {
+    this.handleScroll()
+  }
+
+  handleScroll () {
+    const labelIsAboveContainerTop = this.activeLabel.current?.offsetTop <= this.frameList.current?.scrollTop
+    const labelIsBelowContainerBottom = this.activeLabel.current?.offsetTop >= (this.frameList.current?.scrollTop + this.frameList.current?.clientHeight)
```

Comment:
Added `?` for testing, I think ok per:
![Screen Shot 2020-03-04 at 2 39 18 PM](https://user-images.githubusercontent.com/12118751/75921448-bebc4880-5e26-11ea-9377-a6721da0010b.png)

Reply:
It's stage 4, so should be fine to use!